src/assets/server/rest/v1/schemas/common/common.json

```diff
@@ -112,7 +112,8 @@
     },
     "password": {
       "type": "string",
-      "sanitize": "mongo"
+      "sanitize": "mongo",
+      "pattern": "(?=.*[a-z])(?=.*[A-Z])(?=.*[0-9])(?=.*[!#@:;,<>\/''\\$%\\^&\\*\\.\\?\\-_\\+\\=\\(\\)])(?=.{8,})"
```

Comment:
Not sure the pattern enforce a min length, have you checked?

Reply:
Yes it does

Reply:
And the validator does not does not barf in the PUT case if the password is empty?

Reply:
There is not yet a validator for the user creation/update, it is still handled by the UserSecurity.
But it won't, as we could set the 'password' field as not required in the update scheme